src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
+* Operator and individual applications have separate service accounts with appropriate privileges
+* For Openshift, Lighthouse CoreDNS metrics are integrated with other CoreDNS metrics
```

Comment:
^
@vthapar @skitt  @maayanf24  ^ I think it's just that we provide lighthouse CoreDNS metrics, and it's general not only openshift, right? may be I got it wrong

Reply:
@vthapar @skitt @maayanf24 hey. Can one of you please confirm this?